ANDROIDAUTO.md

```diff
@@ -0,0 +1,80 @@
+# Navigation for Android Auto
+
+This guide explains how to enable and integrate Android Auto with the Flutter Navigation SDK.
+
+## Requirements
+
+- Android device
+- Android Auto test device or Android Automotive OS emulator
+
+## Setup
+
+Refer to the [Android for Cars developer documentation](https://developer.android.com/training/cars) to understand how the Android Auto works and to complete the initial setup. Key steps include:
+
+- Installing Android for Cars App Library.
+- Configuring your app's manifest file to include Android Auto.
+- Declaring a minimum car-app level in your manifest.
+- Creating 'CarAppService' and session
+
+For all the steps above, you can refer to the Android example application for guidance.
+
+### Screen for Android Auto
+
+Once your project is configured accordingly, and you are ready to build the screen for Android Auto, you can leverage the `AndroidAutoBaseScreen` provided by the SDK. This base class simplifies the setup by handling initialization, teardown, and rendering the map on the Android Auto display.
+
+Please refer to the `SampleAndroidAutoScreen.kt` file in the Android example app for guidance.
+
+To customize the Android Auto experience, override the `onGetTemplate` method in your custom AndroidAutoScreen class, providing your own `Template`:
+
+```kotlin
+override fun onGetTemplate(): Template {
+    /** ... */
+    @SuppressLint("MissingPermission") val navigationTemplateBuilder =
+        NavigationTemplate.Builder()
+            .setActionStrip(
+                ActionStrip.Builder()
+                    .addAction(
+                        Action.Builder()
+                            .setTitle("Re-center")
+                            .setOnClickListener {
+                                if (mGoogleMap == null) return@setOnClickListener
```

Comment:
Please check this snippet? 
this line looks weird `return@setOnClickListener`

Reply:
It is suggested/required by the IDE. You can't just do regular `return` here